app/views/layouts/_navbar.html.erb

```diff
@@ -14,8 +14,15 @@
         <ul class="navbar-nav ml-auto">
             <%# https://github.com/heartcombo/devise#controller-filters-and-helpers %>
             <% if user_signed_in? %>
-            <li class="nav-item">
-                <%= link_to (fa_icon 'user', text: current_user.email), edit_user_registration_path, class: 'nav-link'%>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href id="navbarProfileDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  <%= current_user.username %>
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarProfileDropdown">
+                  <%= link_to "Visit my profile", user_path(current_user), class: 'dropdown-item' %>
```

Comment:
Use something like the following for internationalization. Make sure to set the corresponding value in `config/locales/en.yml`.

```suggestion
                  <%= link_to I18n.t('menu.visit_profile'), user_path(current_user), class: 'dropdown-item' %>
```